azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -143,9 +143,9 @@ public long getAgedQueuedFlowSize() {
     long startTime = System.currentTimeMillis();
     // TODO(anish-mal) FetchQueuedExecutableFlows does a lot of processing that is redundant, since
```

Comment:
We can remove this TODO too, since we're already improving the performance!

Reply:
DONE